staging/cert-manager-setup/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: cert-manager-setup
 home: https://github.com/mesosphere/charts
-version: 0.1.7
+version: 0.1.8
 appVersion: 0.10.1
```

Comment:
Has the app version changed at all with our images? Now might be a good time to double check, and perhaps even update the app version if there are some relevant security patches or bug fixes.

Reply:
Not that I am aware of, and if there is i would prefer to do another PR just for that and isolating that process.